buildout.cfg

```diff
@@ -13,6 +13,10 @@ recipe = zc.recipe.egg
 eggs =
     Sphinx
     docutils
+    ZODB
+    j1m.sphinxautointerface
```

Comment:
Member order is configurable either on a directive-by-directive level or globally. The default is alphabetical, I think.